downloader/artifact_downloader_test.go

```diff
@@ -135,16 +135,11 @@ func Test_DownloadAndSaveArtifacts_RetriesFailingDownload(t *testing.T) {
 		{DownloadURL: svr.URL + "/1.txt", Title: "1.txt"},
 	}
 
-	artifactDownloader := NewConcurrentArtifactDownloader(5*time.Minute, log.NewLogger(), nil)
+	artifactDownloader := NewConcurrentArtifactDownloader(5*time.Second, log.NewLogger(), nil)
 	_, err = artifactDownloader.DownloadAndSaveArtifacts(artifacts, targetDir)
 
 	assert.NoError(t, err)
-	// Total request count is:
-	// 1 + 5 = 6
-	// -----
-	// 1 -> initial request to get the content range
-	// 5 -> first request + 4 retries
-	assert.Equal(t, uint64(6), receivedRequestCount.Load())
```

Comment:
With the new retries (we retry every download chunk 5 times, we use a network client which retries every request 4 times and we also retry the whole thing 3 times) the test execution did not finish in 5 minutes.

So in the new version I am simply checking that there were more than one network calls. If there were then the retry kicked in.